cypress/test/campaignTests/AnnouncementHero.cy.js

```diff
@@ -27,13 +27,22 @@ describe('AnnouncementHero Component', () => {
     cy.get('h3').should('exist');
   
   });
-
+/*
   it('should have a link to submit a session', () => {
     // Assert the link
     cy.get('[data-testid="Button-link"]').should('have.attr', 'href', 'https://conference.asyncapi.com/')
       .should('have.attr', 'target', '_blank')
       .contains('Submit a session');
   });
+*/
+
+  it('should have a link and text for the button', () => {
+    mount(<AnnouncementHero />);
+    cy.get('[data-testid="Button-link"]').should('have.attr', 'href', 'https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1/contribute/aacot-london-edition-61313')
+      .should('have.attr', 'target', '_blank')
+      .contains('Get Your Tickets');
+
+   });
```

Comment:
```suggestion
  it('should have a link and text for the button', () => {
    mount(<AnnouncementHero />);
    cy.get('[data-testid="Button-link"]')
      .should('have.attr', 'target', '_blank')
      //making sure link starts with https://
      .should('have.attr', 'href').and('match', /^https:\/\//)
    cy.get('[data-testid="Button-link"]')
      .find('span') // Find the <span> element inside the <a>
      .and('not.be.empty');
   });
```

I tested above on local, will work

I know your works but problem is that in such case we would have to modify test every time we modify hero, which should not be the case, then tests do not make sense

now test only validates:
- if provided href is a link
- if somebody did not forget to provide `text` for it, so we do not have an empty button

Reply:
alright got it